tests/CSSList/DocumentTest.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Sabberworm\CSS\Tests\CSSList;
 
+use Generator;
```

Comment:
Please don't `use` native top-level PHP classes/functions. Instead, please use them like this where they're used:  `\Generator`